connect/runtime/src/main/java/org/apache/kafka/connect/storage/KafkaStatusBackingStore.java

```diff
@@ -218,6 +249,25 @@ private void sendTaskStatus(final TaskStatus status, boolean safeWrite) {
         send(key, status, entry, safeWrite);
     }
 
+    private void sendTopicStatus(final String connector, final String topic, final TopicStatus status) {
```

Comment:
Do we need to pass the connector name and topic name if we could instead just get them from the `TopicStatus` object?

Reply:
Good point. I saw this method is called with a `null` topic status, but didn't put the two together. It's fine as-is.